test/k8sT/DatapathConfiguration.go

```diff
@@ -558,6 +558,24 @@ var _ = Describe("K8sDatapathConfig", func() {
 			}, DeployCiliumOptionsAndDNS)
 			Expect(testPodConnectivityAcrossNodes(kubectl)).Should(BeTrue(), "Connectivity test between nodes failed")
 		})
+		It("Check connectivity with transparent encryption and direct routing with bpf_host", func() {
+			SkipIfIntegration(helpers.CIIntegrationFlannel)
+			SkipIfIntegration(helpers.CIIntegrationGKE)
+			SkipItIfNoKubeProxy()
+
+			privateIface, err := kubectl.GetPrivateIface()
+			Expect(err).Should(BeNil(), "Unable to determine private iface")
+
+			deploymentManager.Deploy(helpers.CiliumNamespace, IPSecSecret)
+			deploymentManager.DeployCilium(map[string]string{
+				"global.tunnel":               "disabled",
+				"global.autoDirectNodeRoutes": "true",
+				"global.encryption.enabled":   "true",
```

Comment:
I think you will need to revert https://github.com/cilium/cilium/commit/8cbd3096a829940035eea95e1e6b10980b07e46e for this test to pass.

Reply:
Yes agree. I must have had a tree without that patch though.Interesting.